docs/verifiable-document/document-data.md

```diff
@@ -4,8 +4,204 @@ title: Defining Document Schema
+
+Move the binary to the directory of your choice (ie `oa-documents`) or add the binary to the PATH variable of your operating system.
+
+### Make binary executable (MacOS/Linux)
+
+In `oa-documents`:
+
+```sh
+chmod +x open-attestation-linux
+```
+
+### Running the CLI
+
+Once the CLI is made executable, you may run it from the terminal directly. You should see the following if it is running correctly.
+
+In `oa-documents`:
+
+```sh
+./open-attestation-linux
+Open Attestation document issuing, verification and revocation tool.
+
+Commands:
+  index.js filter <source> <destination>    Obfuscate fields in the document
+  [fields..]
+  index.js batch <raw-dir> <batched-dir>    Combine a directory of documents
+  [schema]                                  into a document batch
+
+Options:
+  --help     Show help                                                 [boolean]
+  --version  Show version number                                       [boolean]
+
+The common subcommands you might be interested in are:
+- batch
+- filter
+```
+
+## Understanding the OA Document Schema
+
+The OpenAttestation v2.0 defines the shape of data for the `unwrapped document` - the data before the wrapping process. It is defined in [JSON Schema](https://json-schema.org/) format.
+
+The official OpenAttestation v2.0 schema can be found at https://schema.openattestation.com/2.0/schema.json
+
+### Using Online Schema Validator
+
+For this guide, we will be using an online JSON Schema validator to help us write the unwrapped document.
+
+#### Setting up the JSON Schema Validator with OA Schema
+
+Visit https://www.jsonschemavalidator.net/
+
+Paste the contents from https://schema.openattestation.com/2.0/schema.json into the left panel under "Select Schema".
+
+This will setup the JSON schema validator to validate the JSON inputs on the right against the defined schema.
+
+![Validator Preview](/docs/verifiable-document/document-data/validator-preview.png)
+
+If you start editing the JSON data on the right you should see errors if the data does not conform to the OpenAttestation v2.0 schema. A summary of the number of errors is found on top of the right panel and the details of the errors are found below the two panels.
+
+#### Creating unwrapped document
+
+We will now create the data for your Certificate of Completion. Paste the following JSON data into the right panel of the JSON schema validator tool:
+
+```json
+{
+  "name": "OpenAttestation Tutorial Certificate of Completion",
+  "$template": {
+    "name": "COC",
+    "type": "EMBEDDED_RENDERER",
+    "url": "http://localhost:3000"
+  },
+  "recipient": {
+    "name": "John Doe"
+  },
+  "issuers": [
+    {
+      "name": "Demo Issuer",
+      "documentStore": "0x0000000000000000000000000000000000000000",
+      "identityProof": {
+        "type": "DNS-TXT",
+        "location": "demo.openattestation.com"
+      }
+    }
+  ]
+}
+```
+
+We will need to replace a few value from the schema for it to be valid for you:
```

Comment:
not fond of this sentence, but I'm not able to provide a better suggestion ... so just saying I dont like the sentence, sorry :(